src/app/application.h

```diff
@@ -142,6 +143,6 @@ private slots:
 
     void initializeTranslation();
     void processParams(const QStringList &params);
-    void runExternalProgram(const BitTorrent::Torrent *torrent) const;
+    void runExternalProgram(const BitTorrent::Torrent *torrent, const QString targetProgram) const;
```

Comment:
```suggestion
void runExternalProgram(const QString &targetProgram, const BitTorrent::Torrent *torrent) const;
```

Reply:
I use to have the policy of not relocating prior arguments on my projects, with backwards compatibility in mind.
But be it as you wish.